scene/gui/color_picker.cpp

```diff
@@ -1985,6 +2032,13 @@ void ColorPickerButton::_modal_closed() {
 	set_pressed(false);
 }
 
+void ColorPickerButton::_close_modal() {
+	if (popup) {
+		popup->hide();
+	}
+	set_pressed(false);
```

Comment:
This is handled by `_modal_closed()`.

Reply:
I've tried connecting `choose_color_close_requested` to `_modal_closed` instead, but it appears to do nothing (no errors are printed).

Reply:
I mean `popup->hide()` will call `_modal_closed()`, which unpresses the button (didn't test though). The hide itself is ok here.